### PR TITLE
Add Nextcloud, explorer, goBrowser,  SickRage & CouchPotato

### PR DESCRIPTION
**Pull request by CooperBarrett**
Add SickRage & CouchPotato, Automatic torrent/nzb searching, downloading for TV Shows and Movies
Add Nextcloud
Add explorer
Add goBrowser

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,11 +111,13 @@ _Business Intelligence_
 
   * [Alltube](http://www.alltubedownload.net) - A web interface for [youtube-dl](https://github.com/rg3/youtube-dl), a program to download videos and audio from [more than 100 websites](https://rg3.github.io/youtube-dl/supportedsites.html). ([Demo](http://www.alltubedownload.net), [Source Code](https://github.com/Rudloff/alltube)) `GPL` `PHP`
   * [AmIUnique](https://amiunique.org/) - Learn how identifiable you are on the Internet (browser fingerprinting tool). ([Source code](https://github.com/DIVERSIFY-project/amiunique) `MIT` `Java`
+  * [CouchPotato](https://couchpota.to/) - CouchPotato is an automatic Video Library Manager for Movies. Automatic torrent/nzb searching, downloading, and processing at the qualities you want. ([Source Code](https://github.com/CouchPotato/CouchPotatoServer/)) `Python`
```
Comment (nodiscc) [inline README.md]: Missing license

File: README.md [hunk 2]
```diff
@@ -111,11 +111,13 @@ _Business Intelligence_
...
   * [Healthchecks](https://healthchecks.io/) - Django app which listens for pings and sends alerts when pings are late. ([Source Code](https://github.com/healthchecks/healthchecks)) `BSD` `Python`
   * [Huginn](https://github.com/cantino/huginn) - Allows you to build agents that monitor and act on your behalf - `MIT` `Ruby`
   * [Http2pic](https://http2pic.haschek.at/) - Website screenshots/renderer. It uses the wkhtmltox to render websites with various options. ([Source Code](https://github.com/chrisiaut/http2pic)) `Apache 2.0` `PHP/Javascript`
   * [RSS-Bridge](https://github.com/sebsauvage/rss-bridge) - rss-bridge is a PHP project capable of generating ATOM feeds for websites which don't have one. `Public domain` `PHP`
+  * [SickRage](http://sickrage.github.io/) - SickRage is an automatic Video Library Manager for TV Shows. Automatic torrent/nzb searching, downloading, and processing at the qualities you want. ([Source Code](https://github.com/SickRage/SickRage/)) `Python` `Html` `jQuery`
```
Comment (nodiscc) [inline README.md]: Just `Python` as language  
Missing license

File: README.md [hunk 3]
```diff
@@ -475,6 +477,7 @@ Some [Groupware](#groupware) solutions also feature file sharing and synchroniza
 
   * [Git Annex](http://git-annex.branchable.com/assistant/quickstart/) - File synchronization between computers, servers, external drives
   * [Kinto](https://kinto.readthedocs.org) - Kinto is a minimalist JSON storage service with synchronisation and sharing abilities. ([Source Code](https://github.com/Kinto)) `Apache 2.0` `Python`
+  * [Nextcloud](https://nextcloud.com/) - Access & share your files, calendars, contacts, mail & more from any device, on your terms ([Source Code](https://github.com/nextcloud/server)) `AGPLv3` `MIT`
```
Comment (nodiscc) [inline README.md]: Should be `AGPLv3` `PHP`  
Missing a `.` after "terms"

**Comment by nodiscc** [comment]
Thank you @CooperBarrett can you fix the 3 points mentioned in comments?

**Comment by CooperBarrett** [comment]
I made changes, I think it's good.

File: README.md [hunk 4]
```diff
@@ -111,11 +111,13 @@ _Business Intelligence_
...
   * [Healthchecks](https://healthchecks.io/) - Django app which listens for pings and sends alerts when pings are late. ([Source Code](https://github.com/healthchecks/healthchecks)) `BSD` `Python`
   * [Huginn](https://github.com/cantino/huginn) - Allows you to build agents that monitor and act on your behalf - `MIT` `Ruby`
   * [Http2pic](https://http2pic.haschek.at/) - Website screenshots/renderer. It uses the wkhtmltox to render websites with various options. ([Source Code](https://github.com/chrisiaut/http2pic)) `Apache 2.0` `PHP/Javascript`
   * [RSS-Bridge](https://github.com/sebsauvage/rss-bridge) - rss-bridge is a PHP project capable of generating ATOM feeds for websites which don't have one. `Public domain` `PHP`
+  * [SickRage](http://sickrage.github.io/) - SickRage is an automatic Video Library Manager for TV Shows. Automatic torrent/nzb searching, downloading, and processing at the qualities you want. ([Source Code](https://github.com/SickRage/SickRage/)) `GPLv3` `Python/Html/jQuery`
```
Comment (nodiscc) [inline README.md]: Please remove Html/Jquery as we only list server-side languages/platform (it's also obvious that a web app is HTML+Js client side)

File: README.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ See [Contributing](#contributing).
   - [Analytics](#analytics)
   - [Automation](#automation)
   - [Blogging Platforms](#blogging-platforms)
-  - [Bookmarks & Link Sharing](#bookmarks--link-sharing)
+  - [Bookmarks & Link Sharing](#booksickmarks--link-sharing)
```
Comment (nodiscc) [inline README.md]: Nope.

File: README.md [hunk 6]
```diff
@@ -111,11 +111,13 @@ _Business Intelligence_
...
   * [Healthchecks](https://healthchecks.io/) - Django app which listens for pings and sends alerts when pings are late. ([Source Code](https://github.com/healthchecks/healthchecks)) `BSD` `Python`
   * [Huginn](https://github.com/cantino/huginn) - Allows you to build agents that monitor and act on your behalf - `MIT` `Ruby`
   * [Http2pic](https://http2pic.haschek.at/) - Website screenshots/renderer. It uses the wkhtmltox to render websites with various options. ([Source Code](https://github.com/chrisiaut/http2pic)) `Apache 2.0` `PHP/Javascript`
   * [RSS-Bridge](https://github.com/sebsauvage/rss-bridge) - rss-bridge is a PHP project capable of generating ATOM feeds for websites which don't have one. `Public domain` `PHP`
+  * [SickRage](http://sickrage.github.io/) - SickRage is an automatic Video Library Manager for TV Shows. Automatic torrent/nzb searching, downloading, and processing at the qualities you want. ([Source Code](https://github.com/SickRage/SickRage/)) `GPLv3` `Python/Html`
```
Comment (nodiscc) [inline README.md]: Python only.

**Comment by nodiscc** [comment]
Thank you

**Pull request merged**